src/js/tech/flash.js

```diff
@@ -268,6 +268,16 @@ class Flash extends Tech {
   }
 
   /**
+   * Get the last error object
+   *
+   * @return {Object}
+   * @method error
+   */
+  error() {
+    return this.error_ || {};
```

Comment:
Return the error or a black object.
This sounds like where I want to reset `this.error_`.